cmd/crictl/image.go

```diff
@@ -129,6 +130,11 @@ var listImageCommand = &cli.Command{
 			Name:  "no-trunc",
 			Usage: "Show output without truncating the ID",
 		},
+		&cli.BoolFlag{
```

Comment:
I wonder why these flags are not documented in https://github.com/kubernetes-sigs/cri-tools/blob/master/docs/crictl.md?